lib/cb/utils/validator.rb

```diff
@@ -31,10 +31,12 @@ def raise_response_code_errors(response)
       end
 
       def fail_with_error_details(response, error_type)
-        error = error_type.new
+        processed_response = process_response_body(response)
+        error_message = processed_response['errors'] ? processed_response['errors'][0] : nil
```

Comment:
Would there be any issue with returning an empty string instead of nil?  I am assuming the processed_response['errors']] is an array of strings.  The empty string would make for a consistent interface from this method.  However, I don't know the code upstream from this, is nil a more proper response?